src/SelectBox/SelectBox.js

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useEffect } from 'react';
+import propTypes from 'prop-types';
+import {
+  StyledOptionsList,
+  StyledOptionsListItemInnerButton,
+  StyledOptionsListItem
+} from './SelectBox.styles';
+import { StyledCutout } from '../Cutout/Cutout';
+
+const SelectBox = React.forwardRef(function SelectBox(props) {
+  const { options, value, onSelect, width, height } = props;
+  const selectedListItemRef = useRef(null);
+  const listRef = useRef(null);
+
+  const handleKeyDown = event => {
```

Comment:
You can also destruct the `key` property on the function declaration here as well, like:

```const handleKeyDown = ({ key }) => {```

What do you think?

Reply:
Sure.